daemon/daemon_unix.go

```diff
@@ -1151,3 +1154,20 @@ func setupOOMScoreAdj(score int) error {
 	f.Close()
 	return err
 }
+
+func (daemon *Daemon) setupSeccompProfile() error {
+	b, err := ioutil.ReadFile(seccompProfilePath)
+	if err != nil {
+		// TODO(runcom): generate a default seccomp profile if system wide isn't present?
```

Comment:
If we are going to have an internal one, then we should check if the path is "" and use the default.  If the admin specified  seccomppath and the path does not exist, we should error out.